src/condor_schedd.V6/qmgmt.h

```diff
@@ -516,14 +520,18 @@ class JOB_ID_KEY_BUF : public JOB_ID_KEY {
 };
 
 typedef JOB_ID_KEY JobQueueKey;
+#ifdef JOB_QUEUE_PAYLOAD_IS_BASE
+typedef JobQueueBase* JobQueuePayload;
```

Comment:
Not a fan of JobQueuePayload being a pointer type when the other JobQueue... types are not.

Reply:
agree.